README.rst

```diff
@@ -25,6 +25,12 @@ The ``binary_root`` is a required configuration item, it defines where binaries
 live so that when a new binary is POSTed the service will use this path to save
 the binary to.
 
+distributions_root
```

Comment:
why do we need to make this one configurable vs. just using `/etc/distributions/*` ? (or some other location)